pkg/agent/openflow/network_policy.go

```diff
@@ -2270,7 +2270,7 @@ func (f *featureNetworkPolicy) initLoggingFlows() []binding.Flow {
 }
 
 func (f *featureNetworkPolicy) initGroups() []binding.OFEntry {
-	var groups []binding.OFEntry
+	var entries []binding.OFEntry
```

Comment:
why change this?

Reply:
My bad. Legacy changes of the previous commit. Changed.